Bottom: cd0f87540c1a10d81e232315683152793db607fd
Top:    ce683652d194b9d1a04741e075974fe93571d93a
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 19:37:58 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
